src/github.com/couchbase/sync_gateway/db/changes.go

```diff
@@ -322,6 +394,16 @@ func (db *Database) MultiChangesFeed(chans base.Set, options ChangesOptions) (<-
 					return
 				}
 			}
+
+			// Clean up inactive lateSequenceFeeds (because user has lost access to the channel)
+			for channel, lateFeed := range lateSequenceFeeds {
+				if !lateFeed.active {
+					db.closeLateFeed(lateFeed)
+					lateSequenceFeeds[channel] = nil
```

Comment:
any reason to use `lateSequenceFeeds[channel] = nil` over `delete(lateSequenceFeeds, channel)`